bin/compile/element_tree_visitor.dart

```diff
@@ -112,6 +112,11 @@ class EntireAstVisitor extends RecursiveAstVisitor<void> {
   Future<void> visitUnit(String source) async {
     logger.finer('Getting AST for source "$source"');
 
+    final uriSource = Uri.parse(source);
+    if (!uriSource.hasAbsolutePath) {
```

Comment:
This excludes all relative imports, which is bad. We should be looking at changing like 47, or the Uri resolving logic overall.

Reply:
What do you propose?

Reply:
When this happened to me, I got an `InvalidPathResult`. We could use that?

Reply:
I'd rather figure out the root cause of the issue. I'm afraid I don't have anything to propose because I am currently half way up a mountain with very intermittent mobile data.

I'll have a look when I get back to my hotel tonight.